projects/speech_commands/run_experiment.py

```diff
@@ -40,9 +43,24 @@
   projectDir = path.dirname(suite.options.config)
   dataDir = path.join(path.abspath(projectDir), "data")
 
-  if not path.isdir(path.join(dataDir, "speech_commands")):
-    tar = tarfile.open(path.join(dataDir, "speech_commands.tar.gz"))
-    tar.extractall(path=dataDir)
-    tar.close()
+  if "DOMINO_WORKING_DIR" in os.environ:
```

Comment:
Not for tomorrow, but should we use the 'domino' dataset instead of this code?

Reply:
Yes.